nvme.pyx

```diff
@@ -154,7 +154,7 @@ cdef class NvmeDevice(object):
         Preempt the existing `pr_key` with `cur_key`
         '''
         opcode = nvme.nvme_op_codes.nvme_cmd_resv_acquire
-        reservation_acquire_action = (2 & 0x7)
+        reservation_acquire_action = (1 & 0x7)
```

Comment:
While this change looks correct, I wonder whether we could make things clearer (and less error prone), by adding some enums to the .pxd.  For example:

```
# See 7.2, Figure 392 Reservation Acquire - Command Dword 10
ctypedef enum nvme_racqa:
    nvme_racqa_acquire = 0x00
    nvme_racqa_preempt = 0x01
    nvme_racqa_preempt_and_abort = 0x02
```
then this line will become 

`reservation_acquire_action = (nvme_racqa_preempt & 0x7)`

Reply:
Sure, makes sense!